caddy/content.md

```diff
@@ -147,3 +149,5 @@ volumes:
 ```
 
 Defining the data volume as [`external`](https://docs.docker.com/compose/compose-file/compose-file-v3/#external) makes sure `docker-compose down` does not delete the volume. You may need to create it manually using `docker volume create [project-name]_caddy_data`.
```

Comment:
```suggestion
Defining the data volume as [`external`](https://docs.docker.com/compose/compose-file/compose-file-v3/#external) makes sure `docker compose down` does not delete the volume. You may need to create it manually using `docker volume create [project-name]_caddy_data`.
```

Reply:
In fact, now that we are talking about this section as well: May I suggest to delete this paragraph and the parameter `external = true` in the suggested _compose.yaml_  as a `docker compose down` wouldn't actually delete the volume. One would need to provide an addional `-v` or `--volumes` in addition to loose its content. In many cases providing this flag additionally would cause damage to applications, which usually do not mention this in their sample configurations or examples as Docker users usually seem to be aware of the consequences. If you agree @francislavoie  I would simply delete these two parts in another commit.

Reply:
@francislavoie And please review my comment above and give your opinion on that.
Suggestion 1: remove this paragraph and the `external = true` parameter for the volume in the compose.yaml.
Option 2: Apply your suggested changes and leave everything else as is.

Reply:
@francislavoie Please get back to this soon, so we can resolve the issues and merge this PR. :pray:

Reply:
Hmm yeah, sure. We don't specify `external: true` in https://caddyserver.com/docs/running#setup anymore, so I figure we can strip it out here too.

Reply:
Alright, thanks for your look into it.